web/blog/tenzir-v4.18/index.md

```diff
@@ -0,0 +1,105 @@
+---
+title: Tenzir v4.18
+authors: [dominiklohmann]
+date: 2024-07-11
+tags: [release, health-metrics, tql2]
+comments: true
+---
+
+Monitoring Tenzir Nodes is easier than before with [Tenzir
+v4.18][github-release] and its new health metrics.
+
+TODO: Add a title image
+
+<!-- ![Tenzir v4.18](tenzir-v4.18.excalidraw.svg) -->
```

Comment:
@mavam I likely won't have time to work on this tomorrow; currently anticipating a release on Thursday. If you have some bandwidth for the title image or the blog post in general, feel free to just push onto the branch.